auth-web/tests/unit/test-utils/test-data/affiliations.ts

```diff
@@ -121,5 +121,73 @@ export const businesses: Business[] = [
     name: '0871095 B.C. LTD.',
     corpType: { code: CorpTypes.BENEFIT_COMPANY },
     status: 'HISTORICAL'
+  },
+  // request access (invitations)
+  // single
+  {
```

Comment:
put these at the top? That's how we are expecting it to work right? If it conflicts with a lot of other tests feel free to make a second array containing only the affiliations and combine them as needed for your test suite

Reply:
I do the sort of array in the tests to try to mimic the behavior of the system. I would prefer to rewrite this with actual mock calls to the backend (for existing parts too) but it might take too much time.

Also, I updated the existing tests to take into account addition of these after sorting. 
And in my tests, I do test that they are displayed on first two rows.

Reply:
regarding aii naming, I try not to use names like this in variables that are in any other place except anonymous functions or short functions used for sorting/filtering etc... Exactly for the reason you suggested.

I will update it to the longer name, it will be:  affiliationInviteInfo

Reply:
okay sounds good 👍